synopsys-task/src/synopsys-task/tools-parameter.ts

```diff
@@ -243,39 +258,49 @@ export class SynopsysToolsParameter {
 
     // Check and put environment variable for fix pull request
     if (parseToBoolean(inputs.BLACKDUCK_FIXPR_ENABLED)) {
-      console.log("Black Duck Fix PR is enabled");
-      blackduckData.data.blackduck.fixpr = this.setBlackDuckFixPrInputs();
-      blackduckData.data.azure = await this.getAzureRepoInfo();
-    } else {
-      // Disable fix pull request for adapters
-      blackduckData.data.blackduck.fixpr = { enabled: false };
+      if (isPullRequest()) {
+        taskLib.warning(
+          "Black Duck Fix PR is enabled but it will be ignored since it is a PR/MR scan"
```

Comment:
we can align with the similar messages used in action

Reply:
updated accordingly